src/polytopes/chain.jl

```diff
@@ -31,32 +47,7 @@ This function is provided for IO purposes. Most algorithms
 should be written in terms of `nvertices` and `vertices`
 as they are consistent with each other.
 """
-npoints(c::Chain) = length(c.vertices)
-
-"""
-    nvertices(chain)
-
-Return the number of vertices of the `chain`. In the case
-that the chain is closed, the number of vertices is one
-less the total number of points used to represent the chain.
-"""
-nvertices(c::Chain) = npoints(c) - isclosed(c)
-
-"""
-    vertices(chain)
-
-Return the vertices of the `chain`. In the case that the
-chain is closed, the returned vector is circular and can
-be indexed with arbitrarily negative or positive indices.
-"""
-function vertices(c::Chain)
-  if isclosed(c)
-    vs = @view c.vertices[begin:end-1]
-    CircularVector(vs)
-  else
-    c.vertices
-  end
-end
+npoints(c::Chain) = length(c.vertices) + isclosed(c)
```

Comment:
Can you elaborate on why we are doing these changes and removing nvertices and vertices here?

Reply:
This is explained in the PR description:

> `vertices` and `nvertices` still have the same behavior, but they can now rely on the shared implementation in `polytopes.jl`. Do we still want to keep a separate function for them here so we can keep the documentation that is specific to Chain?